build.properties

```diff
@@ -1,4 +1,4 @@
-VERSION=1.1.7a
+VERSION=1.1.7b
```

Comment:
Please do not change version numbers in pull requests.

Reply:
reverted the version 👍